packages/components/src/Components/TagCount/tagCount.scss

```diff
@@ -0,0 +1,13 @@
+.ins-c-tagCount {
+    color: #9a9a9a;
+    display: flex;
+
+    i {
+        padding: 10px;
+        font-size: 150%;
```

Comment:
I guess `var(--pf-global--icon--FontSize--lg)` will be better here.

Reply:
seconded, don't use percentages for font-size